gr-blocks/lib/wavfile_sink_impl.h

```diff
@@ -15,6 +15,9 @@
 #include <gnuradio/blocks/wavfile_sink.h>
 #include <sndfile.h> // for SNDFILE
 
+#define ITEMS_SIZE 1024
```

Comment:
Make these member variable `constexpr`.

Reply:
Do you prefer ```constexpr``` or ```static constexpr```? I'm seeing both in the tree.

Reply:
`static constexpr` is probably best. (static members should have `s_` prefix).

I believe the only difference here is that non-static would possibly waste a byte of memory.